GAE/src/org/akvo/flow/servlet/ReportServlet.java

```diff
@@ -167,27 +168,36 @@ protected RestResponse handleRequest(RestRequest req) throws Exception  {
 
     public static void queueStart(String baseUrl, Report r) {
         Queue queue = QueueFactory.getDefaultQueue();
-        TaskOptions options = getTaskOptions(baseUrl, r);
+        TaskOptions options = getTaskOptions(baseUrl, 1L, r); //First time
```

Comment:
Why is attempt a Long and not an "int"?

Reply:
ReportTaskRequest.attempt is Long so I can handle if it is missing